JASP-Engine/JASP/R/correlation.R

```diff
@@ -401,7 +401,7 @@ Correlation <- function(jaspResults, dataset, options){
       }
       
       result$vsmpr <- JASP:::.VovkSellkeMPR(result$p.value)
-      result$vsmpr <- ifelse(result$vsmpr == "∞", Inf, result$vsmpr)
+      result$vsmpr <- ifelse(result$vsmpr == "\u221E", Inf, result$vsmpr)
```

Comment:
```
> "∞" == "\u221E"
[1] FALSE
```
This would not be desired behaviour, but the code is actually absolete so it can be deleted:  https://github.com/jasp-stats/jasp-desktop/pull/4303#discussion_r494856062